internal/sandbox/sandboxtest/fs.go

```diff
@@ -4,14 +4,13 @@ import (
 	"io/fs"
 	"path/filepath"
 	"runtime"
+	"slices"
 	"testing"
 	"testing/fstest"
 
-	"golang.org/x/exp/maps"
-	"golang.org/x/exp/slices"
-
 	"github.com/stealthrocket/timecraft/internal/assert"
 	"github.com/stealthrocket/timecraft/internal/sandbox"
+	"golang.org/x/exp/maps"
```

Comment:
There is a builtin `maps` package, but it doesn't have the `maps.Keys` function used in this repo.

See:
* https://pkg.go.dev/maps
* https://pkg.go.dev/golang.org/x/exp/maps